packages/core/http/core-http-server-internal/src/http_server.ts

```diff
@@ -516,6 +516,7 @@ export class HttpServer {
   }
 
   private configureRoute(route: RouterRoute) {
+    const optionsLogger = this.logger.get('http', 'server', this.name, 'options');
```

Comment:
I chose to add a new logger to make it easier to filter for these specific log lines

Reply:
If we want to avoid repeating the `http` and `server` part, I think we can define the logger as`this.log.get(this.name, 'options);`.

Reply:
We'd lose the full context around where the log comes from if we remove `http` and `server`, which makes it very hard to analyze the debug-level logs.
Personally, I'd prefer being able to filter logs by a full logger name that keeps some sort of record of where it's coming from.

Reply:
> We'd lose the full context around where the log comes from if we remove `http` and `server`, which makes it very hard to analyze the debug-level logs.

Mind that I'm using `this.log` (instead of `this.logger`). AFAIK, `this.log` is already scoped to `http.server`, calling `this.log.get` calls a _nested_ logger. If we ever change the parent, it all changes together (as opposed to using the root logger `this.logger`). I hope it makes sense.

> Personally, I'd prefer being able to filter logs by a full logger name that keeps some sort of record of where it's coming from.

++ I agree.

Reply:
oh! cool! then `this.log.get('options')` is enough :)

Reply:
done: commit [`fb86412` (#152621)](https://github.com/elastic/kibana/pull/152621/commits/fb86412ba872c2eccda4f106e9e8577a039da940)